src/EntityManagerFactory.php

```diff
@@ -325,6 +326,17 @@ protected function setCustomFunctions(Configuration $configuration)
     /**
      * @param Configuration $configuration
      */
+    protected function setCustomHydrationModes(Configuration $configuration)
+    {
+        $hydratorConfig = $this->config->get('doctrine.custom_hydration_modes');
```

Comment:
For backwards compatability it should be `$this->config->get('doctrine.custom_hydration_modes', []);`

If we don't add the default `[]`, it will throw an error that it can't foreach a `null`